src/apps/watchfaces/OswAppWatchfaceFitnessAnalog.cpp

```diff
@@ -0,0 +1,323 @@
+    hal->gfx()->print(dists);
+    hal->gfx()->setTextCursor(CENTER_X + 12, DISP_H-40);
+    hal->gfx()->print(LANG_WATCHFACE_FITNESS_DISTANCE);
+}
+
+void OswAppWatchfaceFitnessAnalog::drawWatchFace(OswHal *hal, uint32_t hour, uint32_t minute, uint32_t second, bool afterNoon) {
+    // Indices
+    hal->gfx()->drawMinuteTicks(CENTER_X, CENTER_Y, 116, 112, ui->getForegroundDimmedColor(), true);
+    hal->gfx()->drawHourTicks(CENTER_X, CENTER_Y, 117, 107, ui->getForegroundColor(), true);
+    
+    // Hours
+    hal->gfx()->drawThickTick(CENTER_X, CENTER_Y,  0, 16, (int)(360.0f / 12.0f * (hour + minute / 60.0f)), 3, ui->getForegroundColor(), true, STRAIGHT_END);
+    hal->gfx()->drawThickTick(CENTER_X, CENTER_Y, 16, 60, (int)(360.0f / 12.0f * (hour + minute / 60.0f)), 7, ui->getForegroundColor(), true);
+
+    // Minutes
+    hal->gfx()->drawThickTick(CENTER_X, CENTER_Y,  0, 16, (int)(360.0f / 60.0f * (minute + second / 60.0f)), 3, ui->getForegroundColor(), true, STRAIGHT_END);
+    hal->gfx()->drawThickTick(CENTER_X, CENTER_Y, 16, 105, (int)(360.0f / 60.0f * (minute + second / 60.0f)), 7, ui->getForegroundColor(), true);
+
+#ifndef GIF_BG
+    // Seconds
+    hal->gfx()->fillCircleAA(CENTER_X, CENTER_Y, 6, ui->getDangerColor());
+    hal->gfx()->drawThickTick(CENTER_X, CENTER_Y, -16, 110, 360 / 60 * second, 3, ui->getDangerColor(), true);
+    hal->gfx()->drawPixel(CENTER_X, CENTER_Y, 0);
+#endif
+
+/*    
+    hal->gfx()->drawThickLineAA(100, 50, 160, 70, 9, rgb565(250,0,250));
+    hal->gfx()->drawThickLineAA(50, 100, 70, 150  , 10, rgb565(255,0,200));
+    hal->gfx()->drawThickLineAA(160, 70+20, 100, 95, 10, rgb565(250,0,250));
+    hal->gfx()->drawThickLineAA(70, 200, 80, 150  , 10, rgb565(255,0,200));
+    hal->gfx()->drawThickLineAA(120+10, 120 ,120+20, 120+90 , 10, rgb565(255,0,200));
+    hal->gfx()->drawThickLineAA(120-10, 120 ,120-20, 120+90 , 10, rgb565(255,0,200));
+*/
+
+//    hal->gfx()->drawCircleAA(100, 100, 20, 8, rgb565(0,255,0));
+
+//printf("xxxxxxxxxxxxxxxxxxx\n");
+   // hal->gfx()->drawThickLineAA(50, 50, 150, 60, 10, rgb565(255,0,200));
+
+
+//    hal->gfx()->drawCircleAA(100, 100, 20, 4, rgb565(0,255,0), 90+90, 360);
+//    hal->gfx()->drawCircleAA(100, 100, 40, 4, rgb565(0,255,0), 90+90+30, 360);
+//    hal->gfx()->drawCircleAA(100, 100, 60, 4, rgb565(0,255,0), 90+90+60, 360);
+  //  hal->gfx()->drawCircleAA(100, 100, 80, 4, rgb565(0,255,0), 90+90+88, 360);
+
+
+//    hal->gfx()->drawCircleAA(100, 100, 30, 4, rgb565(255,0 ,0), 180+90, 360);
+//    hal->gfx()->drawCircleAA(100, 100, 50, 4, rgb565(255, 0,0), 180+90+30, 360);
+//    hal->gfx()->drawCircleAA(100, 100, 70, 4, rgb565(255,0 ,0), 180+90+60, 360);
+//    hal->gfx()->drawCircleAA(100, 100, 90, 4, rgb565(255,0 ,0), 180+90+88, 360);
+
+
+//    hal->gfx()->drawCircleAA(100, 150, 30, 8, rgb565(255, 250 ,0), 90, 271);
+
+//    for (int i=0; i < 360; ++i)
+  //      printf("xxx %d  %f\n", i, rpx(120,110,i), rpx(120,110,(float)i));
+
```

Comment:
Maybe drop large commented blocks of code? I assume this is from copy-pasting another watchface and not for debugging?